lms/services/ltia_http.py

```diff
@@ -58,6 +58,7 @@ def _get_access_token(self, scopes):
                 "client_assertion": signed_jwt,
                 "scope": " ".join(scopes),
             },
+            timeout=(20, 20),
```

Comment:
This value seems aggressive enough (twice the default) but not crazy. We'll see how many more timeouts we see after.